.github/workflows/morongo_integration.yml

```diff
@@ -0,0 +1,83 @@
+name: Morango Integration Tests
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  morango_integration_tests_sqlite:
+    name: Morango Integration Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9, '3.10', '3.11']
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Cache SQLite3
+        id: cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements/test.txt', 'requirements/base.txt', 'requirements/cext.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+
+      - name: Install dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: |
+          pip install -r requirements/test.txt --upgrade
+          pip install -r requirements/base.txt --upgrade
+          pip install -r requirements/cext.txt --upgrade
+
+      - name: Run pre-test script
+        run: python test/patch_pytest.py
+
+      - name: Run tests with SQLite
+        run: python -O -m pytest kolibri/core/auth/test/test_morango_integration.py
+
+
+  morango_integration_tests_postgres:
+    name: Morango Integration Tests with PostgreSQL
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: test
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+
+      - name: Install dependencies
+        run: |
+          pip install -r requirements/test.txt --upgrade
+          pip install -r requirements/base.txt --upgrade
+          pip install -r requirements/cext.txt --upgrade
+          pip install -r requirements/postgres.txt --upgrade
+
+      - name: Run pre-test script
+        run: python test/patch_pytest.py
+
+      - name: Run tests with PostgreSQL

```

Comment:
This is nearly there - you also need to configure postgres to be used by setting the environment variables in the Github action step's env, see here for the environment variables needed: https://github.com/learningequality/kolibri/blob/develop/tox.ini#L40

Reply:
(all the ones prefixed with KOLIBRI_ that aren't `KOLIBRI_HOME`